starter-code/scripts/views/repoView.js

```diff
@@ -5,16 +5,21 @@
   /* TODO: Let's compile our new template!
        Save the result of invoking Handlebars in this 'repoCompiler' variable
        that we will pass to the append method below. */
-  var repoCompiler;  // Finish the Handlebars method here!
+  Handlebars.registerHelper('formatDate', function(dateToBeFormatted) {
+    return moment(dateToBeFormatted).format('MMMM DD, YYYY');
+  });
+
+  var repoCompiler = Handlebars.compile($('#repo-template').html()); // Finish the Handlebars method here!
 
   repoView.renderRepos = function() {
     $('#about ul').empty().append(
-      reposObj.withTheAttribute('name')  // TODO: experiment changing this attribute!
+      reposObj.withTheAttribute('description')  // TODO: experiment changing this attribute!
       .map(repoCompiler)
     );
   };
 /* TODO: Call the function that loads (or 'requests') our repo data.
     Pass in some view function as a higher order callback, so our repos
     will render after the data is loaded. */
-
-});
+  reposObj.requestRepos(repoView.renderRepos);
```

Comment:
Great work rendering the repos to the page after receiving a response from the Github API!